adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js

```diff
@@ -427,6 +440,23 @@ var AdyenCheckoutHybris = (function () {
             }
         },
 
+        showAfterPay: function (countryCode) {
```

Comment:
can we please rename this to initiateAfterPay to align the naming with other PMs?

Reply:
done